Source/WebCore/platform/graphics/ImageBuffer.cpp

```diff
@@ -211,6 +211,7 @@ static RefPtr<ImageBuffer> copyImageBuffer(Ref<ImageBuffer> source, PreserveReso
 
 static RefPtr<NativeImage> copyImageBufferToNativeImage(Ref<ImageBuffer> source, BackingStoreCopy copyBehavior, PreserveResolution preserveResolution)
 {
+    WTFLogAlways("copyImageBufferToNativeImage");
```

Comment:
Oops! I don't think this was intended for this change.

Reply:
Oops,I thought I reverted that. It's for the task I was debugging.